xml/System.Runtime.InteropServices/DllImportSearchPath.xml

```diff
@@ -137,7 +137,7 @@
       </ReturnValue>
       <MemberValue>2</MemberValue>
       <Docs>
-        <summary>When searching for assembly dependencies, include the directory that contains the assembly itself, and search that directory first. This value is used by the .NET Framework, before the paths are passed to the Win32 <see href="https://go.microsoft.com/fwlink/?LinkId=236091">LoadLibraryEx</see> function.</summary>
+        <summary>When searching for assembly dependencies, include the directory that contains the assembly itself, and search that directory first. When used in NativeAOT, the application's directory is considered the "assembly directory" and searched. This value is used by the .NET Framework, before the paths are passed to the Win32 <see href="https://go.microsoft.com/fwlink/?LinkId=236091">LoadLibraryEx</see> function.</summary>
```

Comment:
The application directory that this edit talks about is different from the application directory that the previous paragraph talks about. 

This paragraph talks about the managed application directory (the directory returned by `AppContext.BaseDirectory`), the previous paragraph talks about the Windows OS notion of the application directory (`LOAD_LIBRARY_SEARCH_APPLICATION_DIR`). The two may or may not be the same. For example, consider what happens for `dotnet myapp.dll`.

Reply:
I think I've addressed this concern. Please let me know.